environment.yml

```diff
@@ -60,85 +65,88 @@ dependencies:
   - shapely
   - siphon
   - xarray
+  - xgcm
```

Comment:
Ping @jbusecke (I also added you to our hub, if you can please test it out and commit your tutorial material to https://github.com/oceanhackweek/ohw-tutorials)

Reply:
Ill work on that tonight. Sorry for the delay!